recipes/nengo-loihi/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "nengo-loihi" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nengo-loihi-{{ version }}.tar.gz
+  sha256: e0787dd3ba4ae3b14cfcc84bd3349d332845838574405914bd00b4949fdede50
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - jinja2
+    - nengo >=3.1.0
+    - packaging
+    - python >=3.5
+    - scipy >=1.2.1
+
+test:
+  imports:
+    - nengo_loihi
+    - nengo_loihi.builder
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.nengo.ai/nengo-loihi
+  summary: Run Nengo models on the Loihi chip
+  license: MIT
```

Comment:
So the license is actually some custom one, not MIT:

> NengoLoihi is made available under a proprietary license that permits using, copying, sharing, and making derivative works from NengoLoihi and its source code for any non-commercial purpose, as long as the above copyright notice and this permission notice are included in all copies or substantial portions of the software.

> If you would like to use NengoLoihi commercially, licenses can be purchased from Applied Brain Research. Please contact info@appliedbrainresearch.com for more information.

Not sure yet what the right identifier is.

Reply:
```suggestion
  license: LicenseRef-NENGO
```

Reply:
Thanks for catching that @dopplershift!